src/dist/static/style.css

```diff
@@ -1,7 +1,29 @@
 .trace-view-container {
     margin: 10px;
+    position: relative;
 }
 
 .trace-view-container > .trace-content {
     margin-top: 10px;
-}
\ No newline at end of file
+    position: relative;
+}
+
+.trace-view-container > .trace-content > .spinner {
```

Comment:
why not use the id?

Reply:
in general, the shorter the css selector chains the better. https://www.c-sharpcorner.com/article/selectors-efficiently-in-css/
I would use just `#spinner`, unless I'm missing something here